infrastructure/modules/azurerm/kubernetes/main.tf

```diff
@@ -36,15 +36,25 @@ resource "azurerm_kubernetes_cluster" "azurerm_kubernetes_cluster" {
 }
 
 locals {
+  # This ID gets used by AKS to name resources, there's no current way of
```

Comment:
maybe we can force this value to a know one (so there's no need to get it later) ?

Reply:
haven't seen a way in the docs to do that - I believe something can be done via a `data` module that fetches the virtual networks, anyway it looks like some behavior that will change as the product evolves